game_view.rb

```diff
@@ -1,9 +1,12 @@
 class GameView
 
-  attr_reader :guessed_letters
+  attr_reader :guessed_letters, :body_array
```

Comment:
I think instead of referencing body_array directly it would be better to have a method clarifying that this value is changed by `remove_hangman_body_part` like `get_current_hangman`.